libraries/AP_HAL_SITL/Scheduler.cpp

```diff
@@ -257,8 +257,8 @@ bool Scheduler::thread_create(AP_HAL::MemberProc proc, const char *name, uint32_
     stack_size += 2300;
     
     pthread_t thread {};
-    uint32_t alloc_stack = MAX(PTHREAD_STACK_MIN,stack_size);
-    
+    const uint32_t alloc_stack = MAX(size_t(PTHREAD_STACK_MIN),stack_size);
```

Comment:
I know I'm late on review, but shouldn't `alloc_stack` actually be `size_t` rather then `uint32_t` ?